config/config_schema.json

```diff
@@ -30,6 +30,11 @@
             "default": "results/",
             "description": "Path to the output directory."
         },
+        "tmp_dir":{
+            "type": "string",
+            "default": "results/inter_files",
+            "description": "Path to the temporary directory storing the intermediate files."
+        },
```

Comment:
I think this may potentially be confusing. Temporary files, in the common usage of the term, are files generated by a single program. Indeed, some of the tools that we will use will generated temporary files, and these will not necessarily be stored in the same location (but rather whatever the `TMPDIR` and `TMP` environment variables point to).